src/ui/Program.cs

```diff
@@ -3,33 +3,70 @@
 using System.Diagnostics;
 using System.Threading;
 using System.Windows.Forms;
+using DiscordRPC;
+using System.Timers;
```

Comment:
Is this been used?

Reply:
was testing other things and committing with that... it's already been removed 👍 thanks for noticing